pretrain/tinyllama.py

```diff
@@ -62,10 +62,10 @@ def setup(
     eval: EvalArgs = EvalArgs(interval=1000, max_iters=100),
 ):
     hparams = locals()
-    if data is None:
-        data = TinyLlama()
+    data = TinyLlama() if data is None else data
+    config = Config(name="tiny-llama-1.1b") if model is None else model
```

Comment:
Currently this uses the default config (not tinyllama) but with a different name. The proper way is

```suggestion
    config = Config.from_name("tiny-llama-1.1b") if model is None else model
```